tests/FakeItEasy.Tests/Creation/FakeOptionsConstraints.cs

```diff
@@ -1,14 +0,0 @@
-namespace FakeItEasy.Tests.Creation
-{
-    using System.Collections.Generic;
-    using System.Linq;
-    using FakeItEasy.Creation;
-
-    public static class FakeOptionsConstraints
-    {
-        internal static IProxyOptions HasArgumentsForConstructor(this IArgumentConstraintManager<IProxyOptions> scope, IEnumerable<object> argumentsForConstructor)
-        {
-            return scope.Matches(x => argumentsForConstructor.SequenceEqual(x.ArgumentsForConstructor), "Constructor arguments ({0})".FormatInvariant(string.Join(", ", argumentsForConstructor.Select(x => x.ToString()).ToArray())));
-        }
-    }
-}

```

Comment:
It wasn't used (see my [first commit](https://github.com/FakeItEasy/FakeItEasy/pull/1001/commits/806217100dfc249e24f89be04a01fa3c55afbbe2)). I preferred to remove it rather than convert the usage. I can submit a separate PR if you like.